src/asynqp/__init__.py

```diff
@@ -19,8 +19,12 @@
 def connect(host='localhost',
             port=5672,
             username='guest', password='guest',
-            virtual_host='/', *,
-            loop=None, sock=None, **kwargs):
+            virtual_host='/',
+            *,
+            loop=None,
+            sock=None,
+            nodelay=True,
```

Comment:
I think, that we should add TCP_NODELAY even on passed socket. This is because TCP_NODELAY is a RPC-specific functionality. On rare cases (I dont even know which) user may want to switch off that. And he will have this opportunity. Fact, that TCP_NODELAY will be set on TCP sockets (include passed in) may be documented.